### ISSUE TITLE
Putting figma assets on github for Canada.ca WET template

### ISSUE THREAD

**Issue by chew16** [wet-boew/wet-boew-styleguide#395]
Do you know if its possible to add a repository of WET assets specifically for prototyping applications such as Figma and Adobe XD? Or add a figma file containing the full library of WET assets used for prototyping? We are currently making a library of WET assets in Figma and it would be helpful to add them here for others that prototype.

Thanks!

**Comment by thekodester** [comment]
Where always excited to have people submit new code!

As for where this code shall exist is a good question, you may want to also post in the main repo labelled wet-beow (gets little more love).

But I’ll tag a few people who can help make that decision: @GormFrank and @duboisp

**Comment by duboisp** [comment]
@chew16 Yes it is possible, send me a email at pierre.dubois at servicecanada dot gc dot ca

**Comment by ljkirby** [comment]
@chew16 -- Apologies if this has been addressed elsewhere, but did we ever open-source any of those WET assets for Figma here or elsewhere? It would be awesome to have access to more prototyping libraries for GoC apps 😃
